static/templates/partials/iframely-widget-card.tpl

```diff
@@ -10,7 +10,7 @@
 
             <!-- IF title -->
             <h4 class="media-heading">
-                <a href="{embed.url}" target="_blank" rel="nofollow" class="one-line">
+                <a href="{embed.meta.canonical}" target="_blank" rel="nofollow" class="one-line">
```

Comment:
`url` isn't set on self-hosted installs. [(example)](https://what.thedailywtf.com:8061/iframely?url=http://bit.ly/1KOeSFC&group=true)

The favicon-with-title format already uses meta.canonical. Should I change it to keep track of the original URL and then give that to the template?